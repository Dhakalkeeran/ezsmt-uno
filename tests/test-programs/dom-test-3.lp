#theory lp {
    linear_term {
    - : 2, unary;
    * : 1, binary, left;
    + : 0, binary, left;
    - : 0, binary, left
    };

    dom_term {
    - : 3, unary;
    + : 3, unary;
    * : 2, binary, left;
    + : 1, binary, left;
    - : 1, binary, left;
    .. : 0, binary, left
    };

    &dom/0 : dom_term, {=}, linear_term, head;
    &logic/1 : linear_term, head
}.

&logic(lia).
node(5).
&dom {(-10); (-4); 3; 7; 15} = at(X) :- node(X).
