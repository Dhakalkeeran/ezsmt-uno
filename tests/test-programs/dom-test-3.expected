node(5) 
at(5)=(- 10) 

node(5) 
at(5)=(- 4) 

node(5) 
at(5)=7 

node(5) 
at(5)=15 

node(5) 
at(5)=3
